srcs/incs/ConnectionIOStatus.hpp

```diff
@@ -0,0 +1,21 @@
+#ifndef SRCS_INCS_CONNECTIONIOSTATUS_HPP_
+#define SRCS_INCS_CONNECTIONIOSTATUS_HPP_
+
+struct ReadRequestStatus {
+	typedef enum Type {
+		kFail,
+		kSuccess,
+		kStart
+	} Type;
+};
+
+struct SendResponseStatus {
+	typedef enum Type {
+		kFail,
+		kSuccess,
+		kCompleteClose,
+		kCompleteKeep
+	} Type;
```

Comment:
As far as I've been able to tell, unlike C, C++ doesn't need the "enum" keyword before the tipe when declaring a variable of the type. As long as it's in the right namespace, the typedef as you have it is unnecessary, because it's already aliased in the first place anyway.

Reply:
TLDR:
```
diff --git a/srcs/incs/ConnectionIOStatus.hpp b/srcs/incs/ConnectionIOStatus.hpp
index b3d351b..9ee5117 100644
--- a/srcs/incs/ConnectionIOStatus.hpp
+++ b/srcs/incs/ConnectionIOStatus.hpp
@@ -2,20 +2,20 @@
 #define SRCS_INCS_CONNECTIONIOSTATUS_HPP_
 
 struct ReadRequestStatus {
-	typedef enum Type {
+	enum Type {
 		kFail,
 		kSuccess,
 		kStart
-	} Type;
+	};
 };
 
 struct SendResponseStatus {
-	typedef enum Type {
+	enum Type {
 		kFail,
 		kSuccess,
 		kCompleteClose,
 		kCompleteKeep
-	} Type;
+	};
 };
 
 #endif  // SRCS_INCS_CONNECTIONIOSTATUS_HPP_
```

Reply:
Good to know. Thanks.